ArduCopter/Parameters.h

```diff
@@ -359,6 +359,7 @@ class Parameters {
         k_param_rtl_climb_min,
         k_param_rpm_sensor,
         k_param_autotune_min_d, // 251
+        k_param_avoidance = 252, // AP_Avoidance
```

Comment:
this may be a good opportunity to jump to using higher indexes (>255) in favor of leaving the easier-to-grasp lower number index for newbie programmers. May be a good rule that no maintainer should use the lower range to make them last longer